src/utils/parseConfig.js

```diff
@@ -0,0 +1,8 @@
+import fs from 'fs';
+import yaml from 'js-yaml';
+import stripComments from 'strip-json-comments';
```

Comment:
`YAML` and `JSON`. `YAML` comments are stripped by parser, but `JSON` is passed as is. With this one users can have comments in their `JSON`ified `.bootstraprc`.